### DISCUSSION TITLE
ActualizarClienteRequest no recibe modelo Cliente

### DISCUSSION THREAD

**Discussion by JuliusLancelot** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#63] [Q&A]
Cómo puedo enviar un modelo Cliente a "ActualizarClienteRequest", en lugar de escribir el nombre de todos los campos de la base de datos?

**Comment by AndresRamos** [comment]
Tendria que crear una nueva solicitud que aceptara el modelo `Cliente`. No es mala idea pero originalmente lo deje para que tu decidieras que datos actualizar como se hace con el SDK.
